cli/resolver.rs

```diff
@@ -291,6 +295,12 @@ impl NpmResolver for CliGraphResolver {
     }
   }
 
+  fn on_resolve_bare_builtin_node_module(&self, module_name: &str) {
+    if !self.quiet {
+      eprintln!("Warning: Resolving \"{module_name}\" as \"node:{module_name}\". If you want to use a built-in Node module, add a \"node:\" prefix.")
```

Comment:
We can get rid of the `quiet` property and just use the `log::warn!` macro. The `--quiet` flag will make that automatically work here.